### PR TITLE
fix: Modify 'load' behavior to append data to existing sheet

### PR DESCRIPTION
**Pull request by ssahaxd**
Details:
- Added logic to detect existing sheet and append data if present.
- There is no direct way to append on the sheet, so I am reading the sheet to get the last row number and then writing after that row number.

Resolves: #5472

# Description

- [x] Summary of the change / bug fix.
- [x] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [x] Relevant motivation and context.
- [ ] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [x] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [x] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.
- [ ] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).  Update any user guides that are affected by the changes.
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [integration test script](https://github.com/OpenBB-finance/OpenBBTerminal/tree/develop/openbb_terminal/miscellaneous/integration_tests_scripts).

# Others
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Review by deeleeramone — CHANGES_REQUESTED** [review]
Thanks for the quick action, and it seems like a good solution.  If you can patch in the ability to create a new sheet when the `sheet-name` is not found, I can give it a thorough check to find edge cases.

File: openbb_terminal/helper_funcs.py [hunk 1]
```diff
@@ -1602,15 +1602,28 @@ def export_data(
...
+                    # Get the number of rows in the existing data
+                    start_row = (
+                        existing_df.shape[0] + 1
+                    )  # Add 1 to start writing after the last row
+
+                    # Append data to the existing sheet
                     with pd.ExcelWriter(
                         saved_path,
                         mode="a",
-                        if_sheet_exists="new",
```
[the remark this answers is not in the document]
Comment (ssahaxd) [reply openbb_terminal/helper_funcs.py:1608]: I have fixed the 'worksheet not found' error. Moreover, I would also like to ask the user, if they provide an existing sheet name, if they want to append or replace it. I was wondering if there is some code that I can reuse to prompt the user and get the input.

[the remark this answers is not in the document]
Comment (deeleeramone) [reply openbb_terminal/helper_funcs.py:1608]: There is a Feature Flag - environment variable - that is called "overwrite".  When this is false, the user is prompted Y/N to overwrite an existing file when the exported filename already exists.  I don't see why that can't be recycled for this purpose as well.

[the remark this answers is not in the document]
Comment (ssahaxd) [reply openbb_terminal/helper_funcs.py:1608]: Changes done.
image

**Review by deeleeramone — APPROVED** [review]
Nice solution, thanks for the contribution!

**Pull request merged**